specs/devexperience/DevExperience.md

```diff
@@ -0,0 +1,181 @@
+
+### 5.1.4 – Helping Developers call the right APIs
+Many WinRT APIs do not work in certain scenarios. Whether they don’t work because of the difference in App model, or whether they require Package Identity, we will help developers where possible, to make the right API calls in their application. We will generate a list of API calls that don’t work in Win32, as well as APIs that require Package identity, and provide tooling for .NET developers through a Roslyn Analyzer.
+Here is the list of APIs that we can use as a starting point: [Windows Runtime APIs available to a packaged desktop app (Desktop Bridge)](https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api)
+
+Prior Art: We started on a Roslyn Analyzer for [incompatible WinUI3 APIs](https://github.com/dotnet/try-convert/blob/feature/winui/src/WinUI.Analyzer/Analyzer/DeprecatedUseAnalyzer.cs).
+
+#### 5.1.4.1 – Analyzer for App Authors
+With the WindowsPackageType property, we can expose the exact type of application the customer is building, and therefore provide the exact type of information that they need.
+
+#### 5.1.4.2 – Analyzer for Library authors
+Since library authors won’t set the WindowsPackageType property, the analyzer for library authors should be general purpose, and provide messages such as “this API behaves differently in Desktop vs UWP, add a runtime check or use <replacement API>”.
+
+### 5.2 – New Apps and WinUI3
+
+#### 5.2.1 - New WinUI3 Apps
+Creating new WinUI3 apps will have the proper manifest files included, and project files by design. Developers will use Visual Studio to create new projects, and .NET developers will have the options of using the .NET CLI. There will be a separate doc more specifically focused on the new project experience for Windows developers that will go into more detail.
+
+##### 5.2.1.1 - Manifest Files
+All new WinUI3 apps will have a package.appxmanifest file, but the SxS manifest should be removed. The tooling will supply default manifest files, which it can use mt.exe to merge. The “default” manifests include:
+-	WinRT registrations for WinUI (and Reunion ) types
+-	PerMonV2 (and other DPI awareness) settings
+The reason for this is that over time, we want to consolidate on a single manifest. This certainly won’t be possible today, and we won’t try to tell that story yet. There are things that can only be accomplished with a SxS manifest, and those scenarios will absolutely still work. Note, that in order to consolidate on a single manifest, we don’t need to change the OS or runtime implementation. We could use the package.appxmanifest format to produce a SxS manifest for unpackaged apps, just like we produce an appxmanifest.xml file for packaged/sparse apps today.
```

Comment:
>The reason for this is that over time, we want to consolidate on a single manifest

What multiple manifests today are you implying? SxS? MSIX? package.appxmanifest? Other?

Reply:
Manifest files that exist in the developers project, that they use to configure their application. We really shouldn't have two, and should move anything that can only be accomplished in a SxS manifest into the Package.appxmanifest file (could use a possible rename).

Reply:
>move anything that can only be accomplished in a SxS manifest into the Package.appxmanifest file

Did you mean anything that can be accomplished via appxmanifest should be moved out of SxS manifest?
(as-is it reads the other way - if both work use SxS. But the other direction makes more sense)